src/components/profile-update/profile-update.jsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import { Form, Button, Card, CardGroup, Container, Col, Row } from 'react-bootstrap';
+import { BrowserRouter as Router, Link } from 'react-router-dom';
+
+import React, { useState } from 'react';
+import axios from 'axios';
+import PropTypes from 'prop-types';
+
+
+
+export function ProfileUpdate(props) {
+
+
+    console.log("props: ", props);
+    const [username, setUsername] = useState('');
```

Comment:
I think if you have the user details already, we should not default the `username` to an empty string, Why don't we instead default it to the `user.username`? What do you think? Same applies to the rest.

Reply:
I thought that current user would like to change their username as well as any other details (pass, email, DOB)

Reply:
Yeah agreed. If I want to update my first name, and am on the first_name input, I would rather it be prepopulated with my name than an empty field. 

Assume the user has to fill in 10 different things. If that user has to re-fill all of them everytime they update their profile then the application would not be user friendly. But if we pre-fill all of them with their details and allow them update only the sections they are interested in, then that's much better.

Does that make sense?  Let me know.

Reply:
Yes, taht's much better idea.

Reply:
having problem in prefilling the fields of the form, 
I'm trying to use :  const [username, setUsername] = useState(user.Username);
I got error Username not defined